### PR TITLE
[Flink AI Flow] Fix start script for code refactor

### PR DESCRIPTION
**Pull request by jiangxin369**
## What is the purpose of the change

 Fix start script for code refactor

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

### CODE REVIEW

File: flink-ai-flow/bin/start_aiflow.py [hunk 1]
```diff
@@ -24,21 +24,32 @@
...
         # type of database backend in master
         db_type: sql_lite
-        # the default notification service is no need to started
-        # when using the airflow scheduler 
-        start_default_notification: False
-        # uri of the notification service
+
+        # whether to start the scheduler service
+        start_scheduler_service: True
+
```
Comment (SteNicholas) [inline flink-ai-flow/bin/start_aiflow.py:43]: Add the `start_default_notification: False`.

File: flink-ai-flow/bin/start_aiflow.py [hunk 2]
```diff
@@ -24,21 +24,32 @@
...
         # Config of master server
 
-        # endpoint of master
+        # endpoint of AI Flow Server
         server_ip: localhost
         server_port: 50051
-        # uri of database backend in master
+
+        # uri of database backend of AIFlow server
         db_uri: sqlite:///{root_dir_path}/aiflow.db
```
Comment (SteNicholas) [inline flink-ai-flow/bin/start_aiflow.py:36]: Use the MySQL uri.

File: flink-ai-flow/bin/start_aiflow.py [hunk 3]
```diff
@@ -24,21 +24,32 @@
 
 
 def create_default_sever_config(root_dir_path):
+    airflow_deploy_path = os.environ["AIRFLOW_DEPLOY_PATH"]
```
Comment (SteNicholas) [inline flink-ai-flow/bin/start_aiflow.py:27]: It's recommented to use argment better than the environment variable.

**Review by SteNicholas — CHANGES_REQUESTED** [review]

File: flink-ai-flow/bin/start_aiflow.py (hunk 1 above)
Comment (jiangxin369) [reply flink-ai-flow/bin/start_aiflow.py:43]: Is `notification_uri: localhost:50052` not enough ?

File: flink-ai-flow/bin/start_aiflow.py (hunk 2 above)
Comment (jiangxin369) [reply flink-ai-flow/bin/start_aiflow.py:36]: OK.

File: flink-ai-flow/bin/start_aiflow.py (hunk 3 above)
Comment (jiangxin369) [reply flink-ai-flow/bin/start_aiflow.py:27]: Okay.

**Review by SteNicholas — APPROVED** [review]

**Pull request merged**